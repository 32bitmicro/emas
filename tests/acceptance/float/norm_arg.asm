lw	r1, 208357.1278956/11.7799
lw	r2, r1 + 13.0/7.0
lw	r3, 77*2.5+1 + r2
