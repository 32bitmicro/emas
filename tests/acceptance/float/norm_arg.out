@ 0x0000 : 0x4040  /  010 000 0 001 000 000  /  16448
@ 0x0001 : 0x4517  /  010 001 0 100 010 111  /  17687
@ 0x0002 : 0x4088  /  010 000 0 010 001 000  /  16520
@ 0x0003 : 0x0001  /  000 000 0 000 000 001  /  1
@ 0x0004 : 0x40d0  /  010 000 0 011 010 000  /  16592
@ 0x0005 : 0x00c1  /  000 000 0 011 000 001  /  193
